### ISSUE TITLE
image annotations flip?

### ISSUE THREAD

**Issue by Cv9527** [Seanlinx/mtcnn#31]
@Seanlinx 
152 line in imdb.py: m_bbox[0], m_bbox[2] = -m_bbox[2], -m_bbox[0]. It seems the box is fliped. does the flip is necessary when training model?

**Comment by Seanlinx** [comment]
@Cv9527 Not necessary, but adding mirrored images will enlarge the training set and improve the performance.

**Comment by Cv9527** [comment]
during training Pnet, for the face class task, how to ignore part label -1 when calculate softmaxloss, for regression task, how to ignore negative data?

**Comment by Seanlinx** [comment]
@Cv9527 The negativemining layer sets the out_grad to "0" where it should be ignored.
